crdesigner/map_conversion/opendrive/opendrive_conversion/converter.py

```diff
@@ -155,7 +155,7 @@ def lane_section_to_parametric_lanes(
                     plane_groups.append(plane_group)
 
                     if hasattr(lane, 'signal_references'):
-                        left_vertices, right_vertices = plane_group.parametric_lanes[0].calc_vertices(0, 0, None)
+                        left_vertices, right_vertices = plane_group.parametric_lanes[0].calc_vertices(0, 0.5, None)
```

Comment:
What's this magic number? Should this be some parameter?

Reply:
It was from the change from `plane.py` where they originally hard coded with 0.5 instead of using min_delta_s, and if you start using min_delta_s you will get a divide by zero error.